test/pylib/suite/python.py

```diff
@@ -62,7 +62,8 @@ async def recycle_cluster(cluster: ScyllaCluster) -> None:
                these if it came from a failed test.
             """
             for srv in cluster.servers.values():
-                srv.log_file.close()
+                if srv.log_file is not None:
+                    srv.log_file.close()
```

Comment:
nit. This functionality should not be there at all. It's a server responsibility to close file handlers on stop().